### PR TITLE
blogs update (4 blogs from dremio)

### PR DESCRIPTION
**Pull request by AlexMercedCoder**
added four blogs from dremio to the blogs page

### CODE REVIEW

**Comment by gaborkaszab** [comment]
Could you please be a bit more specific in the commit message about what this patch holds? E.g. "Add 2 new blogs from Dremio"
Other than that LGTM!

**Comment by AlexMercedCoder** [comment]
> Could you please be a bit more specific in the commit message about what this patch holds? E.g. "Add 2 new blogs from Dremio" Other than that LGTM!

adjusted the name, anything else I need to do, or am I good and it will eventually get merged?

**Comment by gaborkaszab** [comment]
> > Could you please be a bit more specific in the commit message about what this patch holds? E.g. "Add 2 new blogs from Dremio" Other than that LGTM!
> 
> adjusted the name, anything else I need to do, or am I good and it will eventually get merged?

Could you please do the same for the commit msg as well?

**Comment by AlexMercedCoder** [comment]
> > > Could you please be a bit more specific in the commit message about what this patch holds? E.g. "Add 2 new blogs from Dremio" Other than that LGTM!
> > 
> > 
> > adjusted the name, anything else I need to do, or am I good and it will eventually get merged?
> 
> Could you please do the same for the commit msg as well?

wouldn't let me edit the old commit messages since there was newer commits, but I had to add a few more blogs so that commit message reflects and updated the title and pull message.

File: landing-page/content/common/blogs.md [hunk 1]
```diff
@@ -32,6 +50,7 @@ Here is a list of company blogs that talk about Iceberg. The blogs are ordered f
 ### [Apache Iceberg 101 – Your Guide to Learning Apache Iceberg Concepts and Practices](https://www.dremio.com/subsurface/apache-iceberg-101-your-guide-to-learning-apache-iceberg-concepts-and-practices/)
 **Date**: September 12th, 2022, **Company**: Dremio
 
+
```
[the remark this answers is not in the document]
Comment (AlexMercedCoder) [reply landing-page/content/common/blogs.md:53]: fixed

File: landing-page/content/common/blogs.md [hunk 2]
```diff
@@ -24,6 +24,24 @@ disableSidebar: true
 
 Here is a list of company blogs that talk about Iceberg. The blogs are ordered from most recent to oldest.
 
+### [Ensuring High Performance at Any Scale with Apache Iceberg’s Object Store File Layout](https://www.dremio.com/subsurface/ensuring-high-performance-at-any-scale-with-apache-icebergs-object-store-file-layout/)
+**Date**: September 20, 2022, **Company**: Dremio
+
+**Author**: [Alex Merced](https://www.linkedin.com/in/alexmerced/)
```
[the remark this answers is not in the document]
Comment (AlexMercedCoder) [reply landing-page/content/common/blogs.md:30]: fixed

File: landing-page/content/common/blogs.md [hunk 3]
```diff
@@ -24,6 +24,24 @@ disableSidebar: true
...
+### [Introduction to Apache Iceberg Using Spark](https://www.dremio.com/subsurface/introduction-to-apache-iceberg-using-spark/)
+**Date**: September 15, 2022, **Company**: Dremio
+
+**Author**: [Alex Merced](https://www.linkedin.com/in/alexmerced/)
+### [A Hands-On Look at the Structure of an Apache Iceberg Table](https://www.dremio.com/subsurface/a-hands-on-look-at-the-structure-of-an-apache-iceberg-table/)
+**Date**: August 24, 2022, **Company**: Dremio
```
[the remark this answers is not in the document]
Comment (AlexMercedCoder) [reply landing-page/content/common/blogs.md]: fixed

**Review by Fokko — APPROVED** [review]

**Pull request merged**